packages/loader/container-loader/README.md

```diff
@@ -1,6 +1,47 @@
 # @fluidframework/container-loader
 
-<!-- AUTO-GENERATED-CONTENT:START (README_DEPENDENCY_GUIDELINES_SECTION:includeHeading=TRUE) -->
+
+**Topics covered below:**
```

Comment:
Slightly torn to have a TOC and right after things that are not listed in it (the auto-generated stuff) but I think I can live with it. TOC at the start does seem better.